### ISSUE TITLE
1.4.0 Before Release Todos

### ISSUE THREAD

**Issue by mrdokenny** [mrdokenny/Cookie-AutoDelete#57]
I just released 1.4.0b3 to the AMO development channel for wider audience testing. The plan is to wait for about a week before fully releasing on stable channels. In the meantime, no new functionality will be added to 1.4.0, only bug fixes and the below todos. Report any bugs by making a new issue or commenting on the feature tracking issues.

- [x] Visual tweaks via CSS (some parts of UI look off-center)
- [x] Update locales
- [x] Update Screenshots
- [x] Update Descriptions

@Strayer @yfdyh000 @AdmiralAnimE  
There's been a lot of changes to the `extension/_locales/en/manifest.json` file including moving it to a different folder which caused some updates to be lost. I recommend using a diff tool like this [(sample de locale diff with en)](https://www.diffchecker.com/IkRxgSXg) to check what has been changed. 

Basically, make a new fork, copy any missing strings, and update the current strings if the "description" part is different in the locales in any of the strings (also remember to update the description part as well).

In the future, I'll probably move to Crowdin once CAD becomes feature complete and there won't be as many strings to update.

Thanks to all the translators, testers, bug reporters, etc.

**Comment by Strayer** [comment]
Updated German translation in PR #63
